packages/wallet-management/src/wallets.ts

```diff
@@ -255,6 +255,14 @@ const okx: Wallet = new InjectedConnector(
   (window as any).okxwallet,
 );
 
+const rabbywallet: Wallet = new InjectedConnector(
+  {
+    name: 'Rabby Wallet',
+    installed: async () => (window as any).ethereum,
+    icon: walletIcons.rabbywallet,
+  },
+  (window as any).ethereum,
```

Comment:
when going with window.rabby we should pass `(window as any).rabby)` as injection here